src/app/lib/utilities/preprocessor/rules/cpsAssetPage/timestampToMilliseconds/index.js

```diff
@@ -1,33 +1,54 @@
-import pathOr from 'ramda/src/pathOr';
+import path from 'ramda/src/path';
 import deepClone from 'ramda/src/clone';
-import semverCompare from 'semver-compare';
+import compose from 'ramda/src/compose';
 
-const timestampToMilliseconds = jsonRaw => {
-  const firstPublished = pathOr(null, ['metadata', 'firstPublished'], jsonRaw);
-  const lastPublished = pathOr(null, ['metadata', 'lastPublished'], jsonRaw);
-  const rawAresVersion = pathOr(null, ['metadata', 'version'], jsonRaw);
+// ARES sometimes reports timestamps in seconds; sometimes in milliseconds
+// This standardises that by assuming any timestamp before 1973 needs converted to ms
+const MINIMUM_TIMESTAMP_VALUE = 100000000000; // March 1973
 
-  // if cant find ares version, do nothing
-  if (!rawAresVersion) return jsonRaw;
-
-  const aresVersion = rawAresVersion.replace('v', '');
-
-  const semverComparison = semverCompare(aresVersion, '1.1.0');
-
-  // if ares version is 1.1.0 or above, do nothing
-  if (semverComparison !== -1) return jsonRaw;
+const standardiseMetadataTimestamps = jsonRaw => {
+  const firstPublished = path(['metadata', 'firstPublished'], jsonRaw);
+  const lastPublished = path(['metadata', 'lastPublished'], jsonRaw);
+  const lastUpdated = path(['metadata', 'lastUpdated'], jsonRaw);
 
   const json = deepClone(jsonRaw);
```

Comment:
a suggestion without the deep clone with a couple of useful functions that could be used further down

```js
const isInSeconds = timestamp => timestamp < MINIMUM_TIMESTAMP_VALUE;

const standardiseTimestamp = timestamp =>
  isInSeconds(timestamp) ? convertToMilliseconds(timestamp) : timestamp;

const standardiseMetadataTimestamps = jsonRaw => {
  const firstPublishedInMs = standardiseTimestamp(jsonRaw.firstPublished);
  const lastPublishedInMs = standardiseTimestamp(jsonRaw.lastPublished);
  const lastUpdatedInMs = standardiseTimestamp(jsonRaw.lastUpdated);

  return mergeDeepLeft(
    {
      metadata: {
        firstPublished: firstPublishedInMs,
        lastPublished: lastPublishedInMs,
        lastUpdated: lastUpdatedInMs,
      },
    },
    jsonRaw,
  );
};
```

sorry I'm at a loose end this morning!